catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -56,6 +58,8 @@
 
   private final LatLonCoordinateProcessor latLonCoordinateProcessor;
 
+  private final MgrsCoordinateProcessor mgrsCoordinateProcessor = new MgrsCoordinateProcessor();
```

Comment:
❓ Why not instantiate this in the constructor like `latLonCoordinateProcessor` for the sake of consistency?